### ISSUE TITLE
Display fixme value when highlighting issue

### ISSUE THREAD

**Issue by boothym** [openstreetmap/iD#7206]
When iD encounters an object with a fixme tag, it notifies the user by highlighting it as an issue - see screenshot. However, to actually see what the fixme says, the user must scroll down to the bottom of the list of fields. 

The text "Address this concern" is not exactly helpful either, as it is not a link which shows the fixme field. I'm on a typical 1920x1080 screen resolution.

![Screenshot_2020-01-03 OpenStreetMap](https://user-images.githubusercontent.com/16965772/71748022-3858aa00-2e69-11ea-941b-27e520912aa6.png)

I propose that the fixme field is moved up into the section with the yellow issue box, or at least to the very top of the fields dropdown (above "name"). The same could be done for some other issues where the appropriate field is below the fold.

![Screenshot_2020-01-03 OpenStreetMap(1)](https://user-images.githubusercontent.com/16965772/71748278-e6fcea80-2e69-11ea-9471-e71c2a60ee1f.png)

**Comment by quincylvania** [comment]
I agree we should make the connection clearer. I see this as similar to #2836. The message should really appear beside the Fix Me field itself.

Tangentially, these issues would really be better suited as prompts than warnings once we do #6101.
